routes/user/progressUser.js

```diff
@@ -0,0 +1,61 @@
+const { Router } = require("express");
+const {
+  CourseProgress,
+} = require("../../controllers/user/coursesProgressUser");
+
+const router = Router();
+
+/**
```

Comment:
Could you please add a line explaining this commented section?

Reply:
This is not a comment, it is the implementation of the swagger tool to document the back-end